src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/FileStorageService.cs

```diff
@@ -51,6 +51,7 @@ public async Task CopyReleaseFilesToPublicContainer(CopyReleaseFilesCommand copy
             // Slug may have changed for the amendment so also remove the previous contents if it has
             if (copyReleaseFilesCommand.ReleaseSlug != copyReleaseFilesCommand.PreviousVersionSlug)
             {
+                // Delete previously uploaded files
```

Comment:
I make a change here in https://github.com/dfe-analytical-services/explore-education-statistics/pull/1844 to move this outside the FileStorageService in an attempt to keep the FileStorageService more generic and keep the logic to file related tasks like add/move/copy/delete. I hope this is ok.